service/controller/v3/resource/vnetpeering/create.go

```diff
@@ -0,0 +1,10 @@
+package vnetpeering
+
+import (
+	"context"
+)
+
+// ApplyCreateChange is noop. Creation goes through ApplyUpdateChange.
+func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange interface{}) error {
```

Comment:
I am wondering why we not just use the normal `Resource` interface. The logic we want here does not really require the CRUD guidance with current and desired state.

Reply:
I also think with normal `Resource` it would easier to review. I'm not very emotional when it comes to this particular resource because it's going to be removed in the next version, but it shouldn't be much work to rewrite it.